virt/backup_restore/virt-backing-up-vms.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+include::modules/virt-document-attributes.adoc[]
+[id="virt-backing-up-vms"]
+= Backing up virtual machines
+:context: virt-backing-up-vms
+:velero-domain: velero.io
+
+toc::[]
+
+You back up virtual machines (VMs) by creating an OpenShift API for Data Protection (OADP) xref:../../virt/backup_restore/virt-backing-up-vms.adoc#oadp-creating-backup-cr_virt-backing-up-vms[`Backup` custom resource (CR)].
+
+The `Backup` CR performs the following actions:
+
+* Backs up {VirtProductName} resources by creating an archive file on S3-compatible object storage, such as xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc#installing-oadp-mcg[Multicloud Object Gateway], Noobaa, or Minio.
+* Backs up VM disks by using one of the following options:
+
+** xref:../../virt/backup_restore/virt-backing-up-vms.adoc#oadp-backing-up-pvs-csi_virt-backing-up-vms[Container Storage Interface (CSI) snapshots] on CSI-enabled cloud storage, such as Ceph RBD or Ceph FS.
+** xref:../../virt/backup_restore/virt-backing-up-vms.adoc#oadp-backing-up-applications-restic_virt-backing-up-vms[Restic file system backups] on object storage.
+
+[NOTE]
+====
+OADP provides backup hooks to freeze the VM file system before the backup operation and unfreeze it when the backup is complete.
+
+The `kubevirt-controller` creates the `virt-launcher` pods with annotations that enable Velero to run the `virt-freezer` binary before and after the backup operation.
+
+The `freeze` and `unfreeze` APIs are subresources of the VM snapshot API. See xref:../../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virt-about-vm-snapshots_virt-managing-vm-snapshots[About virtual machine snapshots] for details.
+====
+
+You can add xref:../../virt/backup_restore/virt-backing-up-vms.adoc#oadp-creating-backup-hooks_virt-backing-up-vms[hooks] to the `Backup` CR to run commands on specific VMs before or after the backup operation.
+
+You schedule a backup by creating a xref:../../virt/backup_restore/virt-backing-up-vms.adoc#oadp-scheduling-backups_virt-backing-up-vms[`Schedule` CR] instead of a `Backup` CR.
+
+include::modules/oadp-creating-backup-cr.adoc[leveloffset=+1]
+include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+2]
+include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+2]
+include::modules/oadp-creating-backup-hooks.adoc[leveloffset=+2]
+include::modules/oadp-scheduling-backups.adoc[leveloffset=+2]
```

Comment:
I originally decided to have this model as level 2 because it is a backup option. But maybe it should be moved up a level because it involves a different CR. OK, I'll bump up the header level.